docs/GettingRawResponse.md

```diff
@@ -0,0 +1,30 @@
+# Getting Raw Response
+
+Steps for getting the raw response [i.e [Response Object](https://developer.mozilla.org/en-US/docs/Web/API/Response)]
+
+**Initialize the Client**
+
+```typescript
+const options = {
+    authProvider: yourAuthProvider
+};
+const client = MicrosoftGraph.Client.init(options);
+```
+
+**Save request to get the raw response before calling action**
+
+The graph request will be returned for all the method calls except for actions, because they makes call to the server, so have to save the copy of graph request instance before calling actions
+
+```typescript
+const graphRequest = client.api("/me").select("displayName")
+let response = await graphRequest.get();
+```
+
+**Get the raw response**
+
+Use `.getRawResponse()` method to get the raw response
```

Comment:
What does  this do beyond just call fetch?  Would be possible to just do something like,
```
let rawRequest = graphRequest.GetRawRequest();
let rawResponse = await fetch(rawRequest);
```

Reply:
Added ResponseType.RAW which needs to be passed in `.responseType()` get the raw response.
Refer: https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/Middleware/docs/GettingRawResponse.md